src/components/TypeWriter.jsx

```diff
@@ -28,10 +28,12 @@ class TypeWriter extends React.Component {
     const active = this.props.typing;
 
```

Comment:
This condition block here causes setting of `visibleChars` state to fire twice along with the `_handleTimeout` handler. So, clearing `_handleTimeout` is necessary.